CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+4.0.0 (May 2018)
+----------------
+
+- Support for big (tiled) Images (PR #243)
+- Allow loading Polygons from OMERO to add to images (PR #253)
+- Started using test_infra to test listing of figure files (PR #270)
```

Comment:
Move to end of list since it is dev specific
Also to match the rest it should ``Start``